webapp/app/appErrors/appErrors.js

```diff
@@ -3,15 +3,14 @@ import './appErrors.scss'
 import React from 'react'
 import { connect } from 'react-redux'
 import { CSSTransition, TransitionGroup } from 'react-transition-group'
-import Markdown from '../../components/markdown'
-
 import * as R from 'ramda'
 
 import { useI18n } from '@webapp/store/system'
 
-import * as ErrorsState from './appErrorsState'
+import Markdown from '../../components/markdown'
 
-import { closeAppError } from './actions'
+import * as ErrorsState from '../../store/system/serviceError/state'
+import { closeAppError } from '../../store/system/serviceError/actions'
```

Comment:
I think that we are doing in this way:

`import { ServiceSystemErrorState, ServiceSystemErrorActions } from '@webapp/store/system'`

`ServiceSystemErrorActions.closeAppError`